File: src/NotesExamensSeancesBundle/Entity/Seance.php
<?php

namespace NotesExamensSeancesBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Seance
 *
 * @ORM\Table(name="seance")
 * @ORM\Entity(repositoryClass="NotesExamensSeancesBundle\Repository\SeanceRepository")
 */
class Seance
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;
    /**
     * @var int
     * @ORM\ManyToOne(targetEntity="\BlocBundle\Entity\Salle")
     * @ORM\JoinColumn(name="salle",referencedColumnName="id")
     */
    private $salle;

    /**
     * @var int
     * @ORM\ManyToOne(targetEntity="\GestionNiveauxBundle\Entity\matiere")
     * @ORM\JoinColumn(name="matiere",referencedColumnName="id")
     */
    private $matiere;

    /**
     * @var int
     * @ORM\ManyToOne(targetEntity="\UtilisateursBundle\Entity\User")
     * @ORM\JoinColumn(name="professeur",referencedColumnName="id")
     */
    private $professeur;

    /**
     * @var int
     * @ORM\ManyToOne(targetEntity="\GestionNiveauxBundle\Entity\classe")
     * @ORM\JoinColumn(name="classe",referencedColumnName="id")
     */
    private $classe;


    /**
     * @var \DateTime
     *
     * @ORM\Column(name="jour", type="date")
     */
    private $jour;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="debut", type="time")
     */
    private $debut;

    /**
     * @return int
     */
    public function getSalle()
    {
        return $this->salle;
    }

    /**
     * @param int $salle
     */
    public function setSalle($salle)
    {
        $this->salle = $salle;
    }

    /**
     * @return int
     */
    public function getMatiere()
    {
        return $this->matiere;
    }

    /**
     * @param int $matiere
     */
    public function setMatiere($matiere)
    {
        $this->matiere = $matiere;
    }

    /**
     * @return int
     */
    public function getProfesseur()
    {
        return $this->professeur;
    }

    /**
     * @param int $professeur
     */
    public function setProfesseur($professeur)
    {
        $this->professeur = $professeur;
    }

    /**
     * @return int
     */
    public function getClasse()
    {
        return $this->classe;
    }

    /**
     * @param int $classe
     */
    public function setClasse($classe)
    {
        $this->classe = $classe;
    }

    /**
     * @return \DateTime
     */
    public function getDebut()
    {
        return $this->debut;
    }

    /**
     * @param \DateTime $debut
     */
    public function setDebut($debut)
    {
        $this->debut = $debut;
    }

    /**
     * @return \DateTime
     */
    public function getFin()
    {
        return $this->fin;
    }

    /**
     * @param \DateTime $fin
     */
    public function setFin($fin)
    {
        $this->fin = $fin;
    }



    /**
     * @var \DateTime
     *
     * @ORM\Column(name="fin", type="time")
     */
    private $fin;



    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set jour
     *
     * @param \DateTime $jour
     *
     * @return Seance
     */
    public function setJour($jour)
    {
        $this->jour = $jour;

        return $this;
    }

    /**
     * Get jour
     *
     * @return \DateTime
     */
    public function getJour()
    {
        return $this->jour;
    }
}

File: mobile/src/entities/Seance.java


package entities; 

public class Seance {

    private int id;
    private int salle;
    private String salleName;
    private int matiere;
    private String matiereName;
    private int professeur;
    private String professeurName;
    private int classe;
    private String classeName;
    private String jour;
    private String debut;
    private String fin;

    public String getSalleName() {
        return salleName;
    }

    public void setSalleName(String salleName) {
        this.salleName = salleName;
    }

    public String getMatiereName() {
        return matiereName;
    }

    public void setMatiereName(String matiereName) {
        this.matiereName = matiereName;
    }

    public String getProfesseurName() {
        return professeurName;
    }

    public void setProfesseurName(String professeurName) {
        this.professeurName = professeurName;
    }

    public String getClasseName() {
        return classeName;
    }

    public void setClasseName(String classeName) {
        this.classeName = classeName;
    }

    public Seance(int id, int salle, int matiere, int professeur, int classe, String jour, String debut, String fin) {
        this.id = id;
        this.salle = salle;
        this.matiere = matiere;
        this.professeur = professeur;
        this.classe = classe;
        this.jour = jour;
        this.debut = debut;
        this.fin = fin;
        
      /*  ExamenServices eS = new ExamenServices();
        this.salleName = eS.getSalleName(this.salle);
        this.matiereName = eS.getMatiereName(this.matiere);
        this.professeurName = eS.getProfesseurName(this.professeur);
        this.classeName = eS.getClasseName(this.classe); */ 
    }

    public Seance(int salle, int matiere, int professeur, int classe, String jour, String debut, String fin) {
        this.salle = salle;
        this.matiere = matiere;
        this.professeur = professeur;
        this.classe = classe;
        this.jour = jour;
        this.debut = debut;
        this.fin = fin;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getSalle() {
        return salle;
    }

    public void setSalle(int salle) {
        this.salle = salle;
    }

    public int getMatiere() {
        return matiere;
    }

    public void setMatiere(int matiere) {
        this.matiere = matiere;
    }

    public int getProfesseur() {
        return professeur;
    }

    public void setProfesseur(int professeur) {
        this.professeur = professeur;
    }

    public int getClasse() {
        return classe;
    }

    public void setClasse(int classe) {
        this.classe = classe;
    }

    public String getJour() {
        return jour;
    }

    public void setJour(String jour) {
        this.jour = jour;
    }

    public String getDebut() {
        return debut;
    }

    public void setDebut(String debut) {
        this.debut = debut;
    }

    public String getFin() {
        return fin;
    }

    public void setFin(String fin) {
        this.fin = fin;
    }

    @Override
    public String toString() {
        return "Seance{" + "id=" + id + ", salle=" + salle + ", matiere=" + matiere + ", professeur=" + professeur + ", classe=" + classe + ", jour=" + jour + ", debut=" + debut + ", fin=" + fin + '}';
    }
    
    
}